spec/factories/user.rb

```diff
@@ -13,7 +13,7 @@
     password 'secret12345'
 
     after(:create) do |user|
-      user.confirm!
+      user.confirm
```

Comment:
With the new devise version this format is deprecated. This is the new way as per devise documentation